### ISSUE TITLE
Multi-Line Strings include the trailing slash

### ISSUE THREAD

**Issue by hlship** [neuland/jade4j#68]
First off, I'm not sure that Jade itself supports multi-line strings by intention:

https://github.com/visionmedia/jade/issues/1447

But assuming it does, Jade4J messs up slightly:

```
div(attr="foo \
  bar \
  baz")
```

compiles to

```
<div attr="foo \  bar \  baz"></div>
```

... that is, the slashes are being included.

**Comment by chbloemer** [comment]
Works in JadeJS 1.11. It does not work in Jade4J Version 1.0.0.
Added testcase issues/68.jade

**Comment by chbloemer** [comment]
Fixed in Jade 1.2.2
